Data/DataModel/Response.cs

```diff
@@ -1,18 +1,53 @@
-﻿using System;
-using System.Collections.Generic;
-using System.ComponentModel.DataAnnotations.Schema;
-using System.Linq;
-using System.Threading.Tasks;
+﻿using System.ComponentModel.DataAnnotations.Schema;
+using System.Text.RegularExpressions;
 
 namespace ServerCore.DataModel
 {
+    /// <summary>
+    /// The response to a particular submission for a given puzzle
+    /// </summary>
     public class Response
     {
+        private string submittedText = string.Empty;
+
+        /// <summary>
+        /// The Id
+        /// </summary>
         [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
         public int ID { get; set; }
+
+        /// <summary>
+        /// The puzzle the response is for
+        /// </summary>
         public virtual Puzzle Puzzle {get; set;}
+
+        /// <summary>
+        /// True if the submitted text is considered a solution to the puzzle
+        /// </summary>
         public bool IsSolution { get; set; }
-        public string SubmittedText { get; set; }
+
+        /// <summary>
+        /// The submitted text
+        /// </summary>
+        public string SubmittedText
+        {
+            get { return this.submittedText; }
+            set { this.submittedText = FormatSubmission(value); }
+        }
+
```

Comment:
Where is the actual response? "Correct", "Keep going", etc.

Reply:
The response seems to be the Note

Reply:
Compare with the old site at https://puzzleday.research.microsoft.com/2018/Author/SubmissionResponses.aspx

You will see columns named "Submission" and "Response" with, well, the submission and response, and another column named "Note" that is empty.

Reply:
Per discussion at the meeting - there should be a response string property in this class, I must have missed it in the port from the old site.